app/templates/events/event_list.html

```diff
@@ -65,7 +65,6 @@ <h1 class="text-center mt-5">Events List for {{selectedTerm.description}}</h1>
         {% if type != "bonner" %}
             <th>Event Program</th>
         {% endif %}
-        <th scope="col">Invitation</th>
```

Comment:
We need to be careful that we are ending up with the same number of headers and columns in every case. Usually removing the header cell altogether isn't what you want.